pages/loginPage.js

```diff
@@ -0,0 +1,41 @@
+const { Page } = require('./page');
+
+const usernameInput = '#username';
+const passwordInput = '#password';
+const loginButton = 'input[name="login"]';
+const flashNotice = '#flash_notice';
+const flashError = '#flash_error';
```

Comment:
@karenn32 Be careful with such variables. Here its fine, but on the large/middle project, if this locator will be used just once, or we already have element's property in the constructor, then some action method like 'typeSomething' with this property used, like that:

![image](https://github.com/user-attachments/assets/ce1969e2-e9cd-40cd-96a3-d594373c4bda)

In a such cases - there is no need for a variables with the locator.